### PR TITLE
Add Health Check wait strategy

### PR DESCRIPTION
**Pull request by cristianrgreco**
#53

### CODE REVIEW

File: src/container.ts [hunk 1]
```diff
@@ -122,6 +126,16 @@ export class DockerodeContainer implements Container {
       .filter(portsArray => portsArray !== null)
       .map(portsArray => Number(portsArray[0].HostPort));
   }
+
+  private getHealthCheckStatus(inspectResult: ContainerInspectInfo): HealthCheckStatus {
+    // @ts-ignore
```
Comment (cristianrgreco) [inline src/container.ts]: Created https://github.com/DefinitelyTyped/DefinitelyTyped/pull/42548

**Comment by cristianrgreco** [comment]
Thanks @sgronblo, please have another look.

**Comment by sgronblo** [comment]
Thanks, now it's looking good to go to me

**Comment by cristianrgreco** [comment]
Thanks @sgronblo, please also review https://github.com/testcontainers/testcontainers-node/pull/56

**Pull request merged**